packages/amplify-schema-validator/src/validators/enum-is-defined-once.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  DocumentNode,
+  Kind,
+  EnumTypeDefinitionNode,
+} from 'graphql';
+import { ValidationError } from '../exceptions/validation-error';
+
+/**
+     * Enum is defined once
+     *
+     * @param schema graphql schema
+     * @returns true if a enum is defined once
+     */
+export const validateEnumIsDefinedOnce = (schema: DocumentNode): Error[] => {
+  const errors: Error[] = [];
+  const enumTypeDefinitions = schema.definitions.filter(
+    (defintion) => defintion.kind === Kind.ENUM_TYPE_DEFINITION,
+  ) as EnumTypeDefinitionNode[];
+
+  enumTypeDefinitions.forEach((enumTypeDefinition) => {
+    const enumName = enumTypeDefinition.name.value;
+    const enumValues = enumTypeDefinition.values;
+    if (!enumValues) {
+      /* istanbul ignore next */
+      return;
+    }
+    console.log('schema definition --- ', enumValues);
```

Comment:
Probably want to pull out this log line.

Reply:
Removed